ga4gh/client.py

```diff
@@ -41,9 +41,6 @@ def __init__(self, urlPrefix, debugLevel=0, workarounds=[], key=None):
 
         requestsLog = logging.getLogger("requests.packages.urllib3")
         requestsLog.setLevel(logLevel)
-        if self._debugLevel == 0:
-            # suppress warning about using https without cert verification
-            requests.packages.urllib3.disable_warnings()
```

Comment:
My version of urllib3 didn't have disable_warnings, so I thought it was now deprecated. Turns out my environment was messed up, and a re-install fixed it, so I added this back in.